agent/runner.go

```diff
@@ -288,6 +288,10 @@ func (r *Runner) Run(ctx context.Context) error {
 
 		// TODO: find better way to update this state and move it to pipeline to have the same env in cli-exec
 		state.Pipeline.Step.Environment["CI_MACHINE"] = r.hostname
+		state.Pipeline.Step.Environment["CI_PIPELINE_STATUS"] = "success"
+		state.Pipeline.Step.Environment["CI_PIPELINE_STARTED"] = strconv.FormatInt(state.Pipeline.Time, 10)
+		state.Pipeline.Step.Environment["CI_PIPELINE_FINISHED"] = strconv.FormatInt(time.Now().Unix(), 10)
+		// DEPRECATED
 		state.Pipeline.Step.Environment["CI_BUILD_STATUS"] = "success"
 		state.Pipeline.Step.Environment["CI_BUILD_STARTED"] = strconv.FormatInt(state.Pipeline.Time, 10)
 		state.Pipeline.Step.Environment["CI_BUILD_FINISHED"] = strconv.FormatInt(time.Now().Unix(), 10)
```

Comment:
below we have the env "JOB" is this even officialy supported? else we shoul delete it!

Reply:
ok but I think it should be renamed to "Step"

Reply:
Good point. Its again another name for a thing also called: 'step'', 'proc', 'task'.

We have them in the docs (so we would need to keep and deprecate them):

![image](https://user-images.githubusercontent.com/6918444/195907274-d31299ba-1063-44ad-a5f4-eb51a7e08cb3.png)

I would suggest to change that in a new PR.

Reply:
ok avert looking throu this pull -> it's worth it's own pull :/

TODO: create issue and cloase this thread